tests/python_client/testcases/test_compaction.py

```diff
@@ -319,13 +319,11 @@ def test_compact_max_time_interval(self):
         for i in range(ct.compact_segment_num_threshold):
             df = cf.gen_default_dataframe_data(tmp_nb)
             collection_w.insert(df)
+            collection_w.create_index(ct.default_float_vec_field_name, ct.default_index)
```

Comment:
Here create_index should be moved to the position before he above collection_w.compact().

Reply:
> Here create_index should be moved to the position before he above collection_w.compact().

Changed.